scipy/fft/_backend.py

```diff
@@ -125,7 +125,7 @@ def register_backend(backend):
     array([1.+0.j])
     >>> set_global_backend("scipy")  # Restore global backend to default
 
-    """
+    """  # noqa: E501
```

Comment:
any sensible change for line 124?

Reply:
Would putting the comment on its own line and split across lines if necessary?